src/components/controls/PasswordInput/__stories__/PasswordInputStories.scss

```diff
@@ -0,0 +1,7 @@
+.password-input-stories {
```

Comment:
Can be done without styles, see message above

Reply:
deleted this style file